server/src/main/java/org/elasticsearch/snapshots/RepositoryIntegrityHealthIndicatorService.java

```diff
@@ -89,7 +91,13 @@ public HealthIndicatorResult calculate(boolean includeDetails) {
                 Collections.emptyList()
             );
         }
-
+        List<HealthIndicatorImpact> impacts = Collections.singletonList(
+            new HealthIndicatorImpact(
+                2,
+                "Snapshots in corrupted repositories cannot be restored. Data loss is possible.",
+                List.of(ImpactArea.SEARCH)
```

Comment:
Should we add a `BACKUP` impact area?  Maybe the corrupted repository should be severity 1?

I'm also wondering if we should include the repository names in the impact (similarly to how we do it in the `shards_availability` indicator) https://github.com/elastic/elasticsearch/blob/1cdb03ddb1c97b4869de33515305b7daee6cebb7/server/src/main/java/org/elasticsearch/cluster/routing/allocation/ShardsAvailabilityHealthIndicatorService.java#L762

Reply:
Do we have a working definition of impact severity levels? My thinking was that it wasn't severity 1 because the system is still responsive and usable. But it would certainly be terrifying to not have a functioning snapshot repo.

Reply:
Ah not really for now. The severity will be very important when we'll have a global impacts section and we'll possibly only report `severity: 1` (requirements TBD)